static/js/containers/CollectionDetailPage.js

```diff
@@ -7,11 +7,14 @@ import _ from 'lodash';
 import OVSToolbar from '../components/OVSToolbar';
 import Footer from '../components/Footer';
 import VideoCard from '../components/VideoCard';
+import Drawer from '../components/material/Drawer';
 import { actions } from '../actions';
+import {setDrawerOpen} from '../actions/commonUi';
 import { getActiveCollectionDetail } from '../lib/collection';
 
 import type { Collection } from '../flow/collectionTypes';
 import type { Video } from '../flow/videoTypes';
+import type { CommonUIState } from "../reducers/commonUi";
```

Comment:
Let's be consistent with the camel-cased acronyms: `CommonUiState`

Reply:
OK